### ISSUE TITLE
Crash with version 0.1.3

### ISSUE THREAD

**Issue by ottnorml** [pwielgolaski/shellcheck-plugin#8]
### Exception report
```
java.lang.Throwable: Invalid range specified: (1280, 1235); 
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:134)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:234)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:229)
	at com.intellij.openapi.util.TextRange.assertProperRange(TextRange.java:225)
	at com.intellij.openapi.util.TextRange.<init>(TextRange.java:44)
	at com.intellij.openapi.util.TextRange.<init>(TextRange.java:33)
	at com.shellcheck.ShellcheckExternalAnnotator.createAnnotation(ShellcheckExternalAnnotator.java:134)
	at com.shellcheck.ShellcheckExternalAnnotator.apply(ShellcheckExternalAnnotator.java:93)
	at com.shellcheck.ShellcheckExternalAnnotator.apply(ShellcheckExternalAnnotator.java:29)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doApply(ExternalToolPass.java:198)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$700(ExternalToolPass.java:40)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.lambda$null$0(ExternalToolPass.java:162)
	at com.intellij.openapi.application.ReadAction.lambda$run$1(ReadAction.java:53)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:977)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:57)
	at com.intellij.openapi.application.ReadAction.run(ReadAction.java:53)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.lambda$run$1(ExternalToolPass.java:159)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.lambda$runUnderDisposeAwareIndicator$10(BackgroundTaskUtil.java:244)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$3(CoreProgressManager.java:180)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:180)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:264)
	at com.intellij.openapi.progress.util.BackgroundTaskUtil.runUnderDisposeAwareIndicator(BackgroundTaskUtil.java:243)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.run(ExternalToolPass.java:157)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:336)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:326)
	at com.intellij.util.ui.update.MergingUpdateQueue.lambda$flush$1(MergingUpdateQueue.java:276)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:290)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:245)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:232)
	at com.intellij.util.Alarm$Request.runSafely(Alarm.java:356)
	at com.intellij.util.Alarm$Request.run(Alarm.java:343)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:228)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:227)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$100(BoundedTaskExecutor.java:26)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2$1.run(BoundedTaskExecutor.java:200)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:229)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:194)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

### Tool stack
Shellcheck v0.1.3
PhpStorm 2018.3.4
Build #PS-183.5429.47, built on February 8, 2019
macOS 10.13.6

**Comment by pwielgolaski** [comment]
Could you share some sample of issue?
What version of she’ll check you use? I’m asking for shell check binaries.

**Comment by ottnorml** [comment]
Thx for the quick response.
This is the output of `shellcheck -V`:
```
ShellCheck - shell script analysis tool
version: 0.6.0
license: GNU General Public License, version 3
website: https://www.shellcheck.net
```

**Comment by pwielgolaski** [comment]
I can not reproduce it, could you provide smallest snippet that reproduce it.

**Comment by ottnorml** [comment]
When I open this script ([~link~](https://gist.github.com/c840fcb82b45d545b720d992c4c0bb03)), I receive the above posted exception.

**Comment by pwielgolaski** [comment]
I released new version 0.1.4, as soon as it is review by JetBrains it will be available for download.
Please reopen if you still have issue with it

**Comment by ottnorml** [comment]
The exception is gone! Thank you, @pwielgolaski. 😃
